include/mbedtls/ssl.h

```diff
@@ -1112,6 +1122,12 @@ struct mbedtls_ssl_context
     size_t out_msglen;          /*!< record header: message length    */
     size_t out_left;            /*!< amount of data not yet written   */
 
+    unsigned char cur_out_ctr[8]; /*!<  Outgoing record sequence  number. */
+
+#if defined(MBEDTLS_SSL_PROTO_DTLS)
+    uint16_t mtu;               /*!< path mtu, used to fragment outgoing messages */
+#endif
```

Comment:
Should be:
```
#endif /* MBEDTLS_SSL_PROTO_DTLS */
```

Reply:
cc @mpg as it's #1939.

Almost all preprocessor guards in this struct span three lines and don't have a comment in their `#endif`; I'll add all of them in the separate PR.

Reply:
Our [style guide](https://tls.mbed.org/kb/development/mbedtls-coding-standards) states that the closing comment is only needed "if the distance to the opening directive is bigger than a few lines or contains other directives", so I don't think it's required here. I can add it nonetheless as it can't hurt (especially if lines get inserted in the future).

Reply:
Addressed in https://github.com/hanno-arm/mbedtls/commit/bc2498a9ffc3d80816cef82055309eb5ab4f915c.